theme.toml

```diff
@@ -60,12 +60,13 @@ label = "honey"
 "ui.virtual.indent-guide" = { fg = "comet" }
 
 "ui.selection" = { bg = "#540099" }
-"ui.selection.primary" = { bg = "#540099" }
+"ui.selection.primary" = { bg = "#1313BE" }
 # TODO: namespace ui.cursor as ui.selection.cursor?
+"ui.cursor" = { fg = "#8E89E6", modifiers = ["reversed"] }
+"ui.cursor.primary" = { modifiers = ["reversed"] }
```

Comment:
```suggestion
"ui.selection" = { bg = "#5F005C" }
"ui.selection.primary" = { bg = "#540099" }
# TODO: namespace ui.cursor as ui.selection.cursor?
"ui.cursor" = { fg = "#B081CC", modifiers = ["reversed"] }
"ui.cursor.primary" = { modifiers = ["reversed"] }
```

how's this looking? @archseer 

pretty muted, fits well with theme. overall seems nicely distinguishable as well, decent contrast ratio as well (5.34, 4.51). some tweaking can be done but this ain't bad either imo.

<img src="https://user-images.githubusercontent.com/19423063/210152998-fe635585-f854-4611-b318-f6d64ac15b49.png" width=700px/>

Reply:
`"ui.selection" = { bg = "#64005c" }` is also looking good (contrast: 5.17)

![image](https://user-images.githubusercontent.com/19423063/210153466-36358fa2-4f5f-4632-a8f9-2375eb410191.png)

Reply:
I was just watching the video at this comment: https://github.com/texstudio-org/texstudio/issues/2467#issuecomment-1345353969

and, the primary/secondary color _felt_ similar .... but yeah, the background being one extrema (dark or ligh) leaves many other contrasting color options for other stuff, as compared to it being relatively in between-ish.